variables.tf

```
@@ -157,6 +157,12 @@ variable "vpc_id" {
   type        = string
 }
 
+variable "stickiness" {
+  description = "Stickiness session enabled."
+  type        = bool
```

```suggestion
  type        = any
```
I think this should be some type of object but rather than define it what about any?